ServerManager/configfiles/Western_Baltic_Sea_basic/Western_Baltic_Sea_basic_1.json

```diff
@@ -39825,11 +39825,10 @@
         "deadline": 192
       }
     ],
-    "edition_name": "Western Baltic Sea edition",
+    "edition_name": "Baltic Sea edition",
     "edition_colour": "#FF7979FF",
-    "edition_letter": "WB",
+    "edition_letter": "B",
     "region": "balticline",
-    "projection": "+proj=laea +lat_0=52 +lon_0=10 +x_0=4321000 +y_0=3210000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs +type=crs",
```

Comment:
Just double check . So, we want the edition name and letter to be equal to the Baltic one ? 
For the region I can see it has to be the same
I think we should add a projection line if we can ? I think this is important for POV later